src/Libraries/Microsoft.Extensions.AI.Evaluation.Console/Commands/ReportCommand.cs

```diff
@@ -58,6 +83,24 @@ internal async Task<int> InvokeAsync(
         await reportWriter.WriteReportAsync(results, cancellationToken).ConfigureAwait(false);
         logger.LogInformation("Report: {outputFilePath} [{format}]", outputFilePath, format);
 
+        // See the following issues for reasoning behind this check. We want to avoid opening the report
+        // if this process is running as a service or in a CI pipeline.
+        // https://github.com/dotnet/runtime/issues/770#issuecomment-564700467
+        // https://github.com/dotnet/runtime/issues/66530#issuecomment-1065854289
+        bool isRedirected = System.Console.IsInputRedirected && System.Console.IsOutputRedirected && System.Console.IsErrorRedirected;
+        bool isInteractive = Environment.UserInteractive && (OperatingSystem.IsWindows() || !(isRedirected));
```

Comment:
Does this mean we won't open the browser if someone is running evals on linux / macOS?

Reply:
Wondering if it would be better to keep the option off by default. Adding a `-open` / `-view` when running locally does not seem like it would be too cumbersome.

Reply:
I haven't tested it, but this logic is supposed to be cross-platform. So it should open a browser on linux or mac. Maybe I can give it a try later.

Reply:
I lean towards keeping it simple and always requiring explicit opt in - however, I don't feel too strongly about this :)